lib/api/multiObjectDelete.js

```diff
@@ -124,7 +125,7 @@ function _formatXML(quietSetting, errorResults, deleted) {
         if (isDeletingDeleteMarker) {
```

Comment:
we can get rid of this variable and just rely on `deletingDeleteMarker`:
```
if (deletingDeleteMarker) {
             deletedXML.push(
                  '<DeleteMarkerVersionId>',
                deletingDeleteMarker,
                  '</DeleteMarkerVersionId>'
              );
}
```
`deletingDeleteMarker` is only defined if we are deleting the delete marker, if I understand correctly.